features/upload_cita_appointments.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Manual Upload of CITA appointments CSV
+  As a Pension Wise data analyst
+  I want to be able to manually upload the CITA appointments CSV
+  So to process the generated data file
+
+  Scenario: I can upload the appointments data
+    Given I am logged in as a Pension Wise data analyst
+    When I upload the cita.csv file for processing
+    Then I can see the data file scheduled for processing
+
+  Scenario Outline: I upload an invalid file
+    Given I am logged in as a Pension Wise data analyst
+    When I upload the <filename> file for processing
+    Then I get the error "<error>"
+    And the data file is not scheduled for processing
+
+    Examples:
+      | filename          | error                           |
+      | excel.xls         | File must have '.csv' extension |
+      | wrong_headers.csv | Incorrect CSV headers           |
+      | binary.csv        | Incorrect CSV headers           |
+
+  Scenario: I require edit permission to access edit manual appointment summary records
+    Given I am logged in as a Pension Wise user - non analyst
```

Comment:
I'm not sure it's worth being explicit about what you're _not_ here. I'd say just `Pension Wise user` should be fine.